.github/workflows/sdk_utils.yml

```diff
@@ -39,8 +48,18 @@ jobs:
       - name: SDK Tests and Lint
         run: |
           set -eExuo pipefail
-          cd frontend/ic_vetkd_sdk_utils
           npm install
+          pushd backend/canisters/ic_vetkeys_encrypted_maps_canister
+          dfx start --background
+          make mock
+          eval $(make export-cmd)
+          popd
+          pushd backend/canisters/ic_vetkeys_manager_canister
+          dfx start --background
```

Comment:
Is there an advantage/disadvantage to doing `dfx start` twice vs. once for the whole project?